Jenkinsfile

```diff
@@ -192,7 +224,41 @@ pipeline {
                         }
                     }
                 }
-            stage('Fault injection') {
+                stage('Single node cart-master') {
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    steps {
+                        provisionNodes NODELIST: env.NODELIST,
+                           node_count: 1,
+                           snapshot: true
+                        runTest stashes: [ 'CentOS-master-install', 'CentOS-master-build-vars' ],
+                                script: """set -x
+                                    . ./.build_vars-Linux.sh
+                                    CART_BASE=\${SL_PREFIX%/install*}
+                                    NODELIST=$nodelist
+                                    NODE=\${NODELIST%%,*}
+                                    trap 'set +e; set -x; ssh -i ci_key jenkins@\$NODE "set -ex; sudo umount \$CART_BASE"' EXIT
+                                    ssh -i ci_key jenkins@\$NODE "set -x
+                                        set -e
+                                        sudo mkdir -p \$CART_BASE
+                                        sudo mount -t nfs \$HOSTNAME:\$PWD \$CART_BASE
+                                        cd \$CART_BASE
+                                        ln -s /usr/bin/fusermount install/Linux/bin/fusermount3
+                                        pip3.4 install --user tabulate
+                                        nosetests-3.4 --with-xunit --xunit-testsuite-name=master test/iof_test_local.py:Testlocal.test_use_ino"
+                                    exit 0
+                                    """,
```

Comment:
Could this be DRYed out?

If it's quite common between cart and iof even, it could go into pipeline-lib perhaps.